pkg/commands/process/settings/rules/ruby/third_parties/bugsnag/testdata/before_bugsnag_notify_add_metadata.rb

```diff
@@ -0,0 +1,15 @@
+class ApplicationController < ActionController::Base
+  before_bugsnag_notify :add_diagnostics_to_bugsnag
```

Comment:
Need to allow simple_symbol reconciliation with method name reconciliation (which will be done as a separate PR)